install

```diff
@@ -98,10 +104,6 @@ if [[ "${build_relocatable}" == true ]]; then
     fi
 fi
 
-# Instal the pre-commit hook
-bash ./githooks/install
```

Comment:
any reason why the pre-commit hook is being removed?

Reply:
never was there